### PR TITLE
Shruti: commenting on Lab1

### PR DESCRIPTION
**Pull request by runnersaw**

### CODE REVIEW

File: GroceryList/app/src/main/java/com/example/siyer/grocerylist/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                        //Toast.makeText(MainActivity.this, "You clicked edit button", Toast.LENGTH_LONG).show();
+
+                        alertEdit.setMessage("Enter the item");
+                        final EditText editedItem = new EditText(MainActivity.this);
+
+                        editedItem.setInputType(InputType.TYPE_CLASS_TEXT);
+                        editedItem.setText(selectedItem);
+                        alertEdit.setView(editedItem);
+
+                        alertEdit.setPositiveButton("Ok", new DialogInterface.OnClickListener() {
```
Comment (runnersaw) [inline GroceryList/app/src/main/java/com/example/siyer/grocerylist/MainActivity.java:64]: You should put these strings in strings.xml and get them by R.string.ok

File: GroceryList/app/src/main/java/com/example/siyer/grocerylist/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                            public void onClick(DialogInterface dialog, int whichButton) {
+                                toDoList.set(position,editedItem.getText().toString());//onclick after the edited text is entered
+                                itemsAdapter.notifyDataSetChanged();
+                                Toast.makeText(MainActivity.this, "Item has been updated", Toast.LENGTH_LONG).show();
```
Comment (runnersaw) [inline GroceryList/app/src/main/java/com/example/siyer/grocerylist/MainActivity.java:68]: Ooh, nice use of a Toast

File: GroceryList/app/src/main/java/com/example/siyer/grocerylist/MainActivity.java (hunk 1 above)
Comment (cwallac) [inline GroceryList/app/src/main/java/com/example/siyer/grocerylist/MainActivity.java:64]: you would have to use getString(R.string.ok)

R.id.ok is actually an integer that getStrings knows how to process to return a String

Alt + Enter -> Extract string resource should add the code for you by default.

**Comment by runnersaw** [comment]
Good job. It looks like it should all work as expected. My only comment is that your code looks like one huge chuck of commands. To make it more readable, you should separate blocks of code into functions and comment blocks of code so readers know what that block is doing.

File: GroceryList/app/src/main/java/com/example/siyer/grocerylist/MainActivity.java (hunk 1 above)
Comment (cwallac) [inline GroceryList/app/src/main/java/com/example/siyer/grocerylist/MainActivity.java:64]: My bad, I thought you said. R.id.ok. I was not paying attention. 
R.string.ok would work to get the string.
getString(R.id.ok) would also work, but what I wrote above
getString(R.string.ok) would not work to get the string

**Pull request closed without merging**